bin/node-template/src/cli.rs

```diff
@@ -90,7 +90,9 @@ where
 
 	let _ = exit_send.send(());
 
-	runtime.block_on(handle);
+	if let Err(e) = runtime.block_on(handle) {
```

Comment:
```suggestion
	let _ = runtime.block_on(handle);
```

As it is only the informant, we can ignore what happens to the task IMHO.

Reply:
It's the `exit` handle as well. I think it's better to log something than to just swallow the error.

Reply:
`let service_res = runtime.block_on(select(service, exit));`

`exit` is also taken here ;)